plugins/gateway/apigeex/README.md

```diff
@@ -0,0 +1,34 @@
+# APIClarity Apigee-X plugin
+
+This is a Apigee-X plugin that integrates with APIClarity.
+For more information about Apigee-X please visit: https://cloud.google.com/apigee/docs
+
+
+## Prerequisites
+
+* APIClarity backend is running and exposed externally through a publicly reachable URL
+* There is an Apigee-X proxy configured in your GCP account and such Apigee-X have connectivity to the above APIClarity
+
+## Install Instructions
+
+1. **Add Certificate to Apigee-X**
+
+    1. In Apigee-x console, go to Admin->Environments->TLS Keystores and create a keystore named `Apiclarity`
+    1. Create an Alias named `Apiclarity` and upload the Apiclarity public certificate
```

Comment:
Section numbers should be incremented

Reply:
In MD you don't need to increment, as MD takes care of it. But I can do it for readability